mk/docker.mk

```diff
@@ -19,7 +19,10 @@ KUMA_IMAGES = $(call build_image,$(IMAGES_RELEASE) $(IMAGES_TEST))
 export DOCKER_BUILDKIT := 1
 
 # add targets to build images for each arch
-# $(1) - GOOS to build for
+# $(1) - GOARCH to build for
+# (TODO): Donot hardcode "linux" platform for images
+# (TODO): May be support other image platforms using argument
```

Comment:
Hmm I think any non linux platform would require signicant rewrite of this so I think these TODOs are not required (the comment fix is right though :) )

Reply:
done